drivers/sdl/joypad_sdl.cpp

```diff
@@ -0,0 +1,388 @@
+/**************************************************************************/
+/*  joypad_sdl.cpp                                                        */
+/**************************************************************************/
+/*                         This file is part of:                          */
+/*                             GODOT ENGINE                               */
```

Comment:
Are we changing references to Godot in new code?

Reply:
There is a PR that updates the copyright disclaimer script. That probably should be executed by maintainers at some point. Also, this PR has been rescued from Godot purgatory, it seems.